### DISCUSSION TITLE
An iOS version

### DISCUSSION THREAD

**Discussion by mcpeapsUnterstrichHD** [Alex313031/thorium#328] [Ideas]
I would like to have an iOS version of this browser that would be killing it

**Comment by gz83** [comment]
@midzer @Alex313031

**Comment by midzer** [comment]
Let me refer to https://chromium.woolyss.com/#ios

**Comment by mcpeapsUnterstrichHD** [comment]
Yeah I know but I mean an chrome app on iOS in the thorium way

**Comment by Alex313031** [comment]
@mcpeapsUnterstrichHD Theres no point. Since it will just be a skin on top of webkit, it won't have any of Thorium's compiler optimizations, and would basically just be Chromium with a different name and logo. Not to mention that I'm not paying the exorbitant fee to put it on the App Store. The only way one could install it is by putting your device in developer mode, and sideloading the app with XCode.
